### PR TITLE
Translation of Step1 - BasicKnowledge 00 前言.md

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
Half done.
Please translate all file names.
谢谢

### CODE REVIEW

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
 
-在互联网发达的今天，很多知识都可以从网络上找到，但是网络上的博客、文章的质量参差不齐，或者重点不明确，或者直接把别人的博客抄袭过来。这种状况使得广大的初学者们学习起来很困难，甚至误入歧途，增加了学习曲线的陡峭程度。当然也有很多博主非常非常负责任，文章质量很高，只是连续度不够，正看得过瘾的时候，没有后续章节了，无法形成知识体系。
+Beginners can choose to read textbooks or books on theory, but the chicken and the egg problem arises: If you don’t understand, then after reading you still won’t understand the theory; if you understand, then you don’t need to read the theory. This is a shortcoming of some textbooks and theoretical books.
 
-初学者也可以选择看一些教材或者理论书籍，但是，一个鸡生蛋蛋生鸡的问题出现了：如果你不懂，那么看完了理论你还是不会懂；如果你懂了，那么你就没必要看理论。这也是很多教材或者理论书籍的缺憾。
+The author has seen professor Andrew Ng's (Wu Enda) classes. The theoretical knowledge is simple yet deep, and is very clear. Although there are very few code examples, I still strongly recommend looking into Andrew Ng's classes. The author's experience is that Andrew Ng's videos can be saved on a mobile device, and you can watch and learn on your own time.
```
Comment (marko-polo-cheno) [inline A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md:12]: Why do you mention this source and do not add a hyperlink so people know what you are talking about?

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
 
-笔者也看过吴恩达老师的课，理论知识讲得由浅入深，还是非常清楚的，虽然代码示例基本没有，但仍然强烈建议大家去看。笔者的心得是：视频可以事先缓存在手机中，利用一些时间片段就可以学习了。
+There are external reasources that you can use to study deep learning online. The author took part in several group purchases. The teachers and teaching assistants are generally very responsible. Lastly, you can rewatch the videos and download the powerpoint course materials. These courses mainly focus on engineering projects, and explain the use of deep learning frameworks and tools. That is, teaching you how to use helpful tools for modeling, training, etc. But for beginners, understanding a new concept may require a lot of previously attained knowledge. An extremely steep learning curve can be frustrating. Maybe someone knows X but doesn't know why. They could end up as an assistant engineer, and their career development ends up being restricted.
 
-社会上还有一些网课，在线讲解深度学习的知识，笔者也参加了几个团购，老师和助教一般都很负责任，最后可以回看录像，下载PPT课件。这些课程一般偏重于工程项目，讲解深度学习框架和工具的使用，即教大家如何使用工具建模、训练等等，也是很有帮助的。但对于初学者来说，理解一个新概念可能需要前面很多个已有知识点的支撑，门槛过高，一下子就变得很沮丧。或者是知其然而不知其所以然，最后沦为调参工程师，职业发展受到了限制。
```
Comment (marko-polo-cheno) [inline A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md:16]: 笔者也参加了几个团购
I did not understand this.

**Comment by marko-polo-cheno** [comment]
@xiaowuhu Who is 'we'?
image

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md (hunk 2 above)
Comment (xiaowuhu) [reply A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md:16]: to buy the online course with others together, so the price will be lower

**Comment by xiaowuhu** [comment]
> @xiaowuhu Who is 'we'?
> image

'we' have 2 students who are working on the translation work, but the progress is not fast. if you want to join us, please let me know. We will pay for the job.

**Review by xiaowuhu — APPROVED** [review]

**Comment by marko-polo-cheno** [comment]
> > @xiaowuhu Who is 'we'?
> > image
> 
> 'we' have 2 students who are working on the translation work, but the progress is not fast. if you want to join us, please let me know. We will pay for the job.

I'd love to. Please email me at markzchen1[at]outlook[dot]com

**Pull request merged**